### ISSUE TITLE
Use sqlite to track the repos and whether or not you have contributed to them

### ISSUE THREAD

**Issue by sharunkumar** [arunsathiya/set-output-janitor#3]
Like, after raising a pr, you can have a column to track it. URLs and everything.

**Comment by arunsathiya** [comment]
Makes sense to me! At this time, I am just remembering organizations that I have patched manually, and have some logic to bail when there's already a PR for that repository. 

https://github.com/arunsathiya/set-output-janitor/blob/aa41178714f023a07dbe55e1f44516a310a9e0eb/main.go#L53-L61

Database to store and check PRs can be an added guard. 👍🏼
